src/edtr-io/plugins/paste-hack/editor.tsx

```diff
@@ -0,0 +1,130 @@
+import { useScopedStore } from '@edtr-io/core'
+import {
+  getParent,
+  insertChildBefore,
+  serializeDocument,
+  removeChild,
+} from '@edtr-io/store'
+import clsx from 'clsx'
+import { either as E } from 'fp-ts'
+import * as t from 'io-ts'
+import { useRef } from 'react'
+
+import { PasteHackPluginProps } from '.'
+import { showToastNotice } from '@/helper/show-toast-notice'
+
+const StateDecoder = t.strict({
+  plugin: t.literal('rows'),
+  state: t.array(
+    t.strict({
+      plugin: t.union([
+        t.literal('article'),
+        t.literal('articleIntroduction'),
+        t.literal('geogebra'),
+        t.literal('anchor'),
+        t.literal('video'),
+        t.literal('table'),
+        t.literal('serloTable'),
+        t.literal('highlight'),
+        t.literal('injection'),
+        t.literal('layout'),
+        t.literal('multimedia'),
+        t.literal('spoiler'),
+        t.literal('important'),
+        t.literal('blockquote'),
+        t.literal('box'),
+        t.literal('image'),
+        t.literal('text'),
+        t.literal('equations'),
+      ]),
+      state: t.unknown,
+    })
+  ),
+})
+
+export const PasteHackEditor: React.FunctionComponent<PasteHackPluginProps> = (
+  props
+) => {
+  const store = useScopedStore()
+  const textareaRef = useRef<HTMLTextAreaElement>(null)
+
+  function throwError(error?: unknown) {
+    showToastNotice('⚠️ Sorry, something is wrong with the data.', 'warning')
+    // eslint-disable-next-line no-console
+    console.error(error)
+    throw new Error(
+      'JSON input data is not a valid edtr-state or contains unsupported plugins'
+    )
```

Comment:
```suggestion
    throw new Error(
      'JSON input data is not a valid edtr-state or contains unsupported plugins', { cause: error }
    )
```

Can you check whether it works?

Reply:
Can't make it work.
Try e.g. `throw new Error('the error', {cause: "test string"})` in browser console.